### ISSUE TITLE
Validation fails with a compression-error

### ISSUE THREAD

**Issue by ethho** [frictionlessdata/frictionless-py#616]
# Overview

Commits 1528d00f822551e28be969195ec75499d6c12496 and more recent, validation using CLI fails: 
```bash
$ frictionless validate --nopool datapackage.json
---
invalid: datapackage.json
---
=================  =========================================================================================
code               message
=================  =========================================================================================
compression-error  The data source could not be successfully decompressed: compression "no" is not supported
=================  =========================================================================================
```

Validation passes without errors on 5bd794179dcbaab080e1e0d0e692b6c875e44d43.

---

Please preserve this line to notify @roll (lead of this repository)

**Comment by roll** [comment]
@eho-tacc,

Thanks for the quick feedback I will release a fix soon

**Comment by roll** [comment]
FIXED in `frictionless@4.0.0a7`

**Comment by lwinfree** [comment]
yes it is working for me now!
